### PR TITLE
added skip-for-now button

### PR DESCRIPTION
**Pull request by tkevans**
added skip-for-now button on "addUserInfoController", so new users don't have to provide personal information before they make a decision (if they don't want to).

### CODE REVIEW

**Comment by tkevans** [comment]
like this pic shows:
screen shot 2018-05-08 at 2 50 10 pm

File: FoodHaus_App/project-swift/Controller/Register/AddUserInfoViewController.swift [hunk 1]
```diff
@@ -50,4 +50,9 @@ class AddUserInfoViewController: UIViewController {
             self.present(alertController, animated: true, completion: nil)
         }
     }
+    
+    @IBAction func saveForLaterButton(_ sender: Any) {
```
Comment (bojms45) [inline FoodHaus_App/project-swift/Controller/Register/AddUserInfoViewController.swift]: Nice work, is this button for the user to skip fill in their profile?

**Comment by wenjunzhu1991** [comment]
good

File: FoodHaus_App/project-swift/Controller/Register/AddUserInfoViewController.swift (hunk 1 above)
Comment (tkevans) [reply FoodHaus_App/project-swift/Controller/Register/AddUserInfoViewController.swift]: yes

**Pull request merged**